tests/middleware/test_message_state.py

```diff
@@ -0,0 +1,34 @@
+import remoulade
+from remoulade.state.message_state import MessageState
+
+
+class TestMessageState:
+    """Class to test the middleware
+    MessageState
+    """
+
+    @staticmethod
+    @remoulade.actor
+    def add(x, y):
+        return x + y
+
+    def test_pending_state_message(self, rabbitmq_broker_state, redis_state_backend):
```

Comment:
I prefer tu use stub broker to test because we are not testing rabbitmb and the RMQ tests don't work on circle ci, But I don't see why we could'nt do both.

Can you add a pytest fixture a bit like `result_backends` which yields both brokers ?

Reply:
name it `brokers`